deltachat-ios/Controller/AccountSetup/Instand Onboarding/InstantOnboardingViewController.swift

```diff
@@ -221,22 +218,16 @@ class InstantOnboardingViewController: UIViewController {
         present(alertController, animated: true)
     }
 
-    @objc private func showMenu(_ sender: Any) {
-        let sheet = UIAlertController(title: nil, message: nil, preferredStyle: .safeActionSheet)
-
-        let showProxySettings = UIAlertAction(title: String.localized("proxy_settings"), style: .default) { [weak self] _ in
-            self?.showProxySettings(sender)
-        }
-
-        let cancelAction = UIAlertAction(title: String.localized("cancel"), style: .cancel)
-
-        sheet.addAction(showProxySettings)
-        sheet.addAction(cancelAction)
-
-        present(sheet, animated: true)
+    private func moreButtonMenu() -> UIMenu {
+        let actions = [
+            UIAction(title: String.localized("proxy_use_proxy"), image: UIImage(systemName: "shield")) { [weak self] _ in
+                self?.showProxySettings()
+            },
```

Comment:
Probably more of a UX-thing, but if we only have one entry a menu, does it make a lot of sense to show that menu -- and not the menu-entry itself? In other words: Why does the BarButtonItem show a menu and not the Proxy-button?

It feels like an extra step for the user to access the proxy-settings.

Reply:
> but if we only have one entry a menu, does it make a lot of sense to show that menu 

yes - as the symbol is not self-explaining for most users. therefore, an additional descriptive text is better.

when proxy is then enabled, the icon is shown directly  - this is also what is done in the main app.